doc/examples/linear_fascicle_evaluation.py

```diff
@@ -63,13 +63,19 @@
                                        line_colors(candidate_sl))
 cc_ROI_actor = fvtk.contour(cc_slice, levels=[1], colors=[(1., 1., 0.)],
                             opacities=[1.])
-vol_actor = fvtk.slicer(t1_data, voxsz=(1.0, 1.0, 1.0), plane_i=[40],
-                        plane_j=None, plane_k=[35], outline=False)
+
+vol_actor = fvtk.slicer(t1_data)
+
+vol_actor.display(40, None, None)
+vol_actor2 = vol_actor.copy()
+vol_actor2.display(None, None, 35)
```

Comment:
Doesn't worth it in this very specific case. The old slicer function was very limited. And even if we replace it with a new one of a different name the signature will be also different. So, no backwards compatibility again. If you want I can make a simple widget to show the 3 slices interactively. But this again will not be exactly backwards compatible because it will be a widget and not an actor. So, I would suggest to merge this PR as is and reflect in the next days if you really want this backwards compatibility with the old function. For me I prefer that this breaks at people's faces as soon as possible so that they start using the new much improved version.